### DISCUSSION TITLE
Polybar for Windows 🪟

### DISCUSSION THREAD

**Discussion by Dhyfer1** [polybar/polybar#2893] [General]
Hi.

Are there any plans for Polybar to have support for the Windows operating system? If they can't add support, what other alternative to Polybar can you recommend that will work with Windows?

**Comment by patrick96** [comment]
No, there will be no windows support. I think Rainmeter can do a lot of stuff, apparently there is even a polybar theme for it

**Reply by Dhyfer1** [reply]
> No, there will be no windows support.

Oh, what a shame to hear that 😕. It would have been great Polybar on Windows, ok, I'll try Rainmeter.
Cheers
